tests/test_buggy_attrs.py

```diff
@@ -101,3 +101,14 @@ def p(self):
         if pattr.name == "p":
             assert category_match(pattr.category, AttrCategory.DESCRIPTOR)
             assert pattr.doc == ("@property with getter, setter, " "deleter, this is p")
+
+
+class A:
+    __slots__ = ['a']
+
+
+def test_slots():
+    a = A()
+    assert len(pdir(a).public.pattrs) == 0
```

Comment:
Makes sense. Do you feel it's necessary to add a new category called `slots`? It might confuse users who are not so familiar with __slots__. Besides, only this can we return something reasonable in [`return dummy`](https://github.com/laike9m/pdir2/blob/b186f8d4fb2d29ad8437bca78b7816c9c2ddbc44/pdir/api.py#L195)

Reply:
I've thought about adding a new category, the problem would be: what if the new category `slotted properties` is contradictory with existing category, like this:
```
>>> class T:
...     __slots__ = ['foo', '__mul__']

>>> pdir(T)
slotted property:
    foo
arithmetic:
    __mul__
```
In my mind, `arithmetic` definitely tells more things to users than `__slots__`, cause `__slots__` is basically an optimization. Now, if we choose to put `__mul__` in `arithmetic`, it is not consistent with `foo`, which would confuse users.

I'm not against tagging slotted properties, but I think there are better ways to do it, one of which could be:
>>> class T:
...     __slots__ = ['foo', '__mul__']

>>> pdir(T)
property:
    foo(slotted)
arithmetic:
    __mul__(slotted)
```

About the second point, I was actually thinking about(before seeing your comment) changing the way `dummy_object` is used. Cause it feels like `dummy_object` should be a place holder for attribute that **exists**(though we are not able to get the actual object), rather than marking "non-existent". I'm still thinking about this problem and might do a commit soon, please don't change this behavior before that.

Reply:
I pushed a new [commit](https://github.com/laike9m/pdir2/commit/47791841d2ddb2172b3b6634516f6d78e0e9902c) to remove unreasonable usage of `dummy_object`. I believe you can now use `dummy_object` in the correct way.